.github/.clear-target-files

```diff
@@ -0,0 +1,2 @@
+**/*
```

Comment:
I figured putting this under `.github` since it's specifically CI related was fine.